src/pages/Dashboard/Activities/UploadImage/index.tsx

```diff
@@ -0,0 +1,48 @@
+import { useState } from 'react';
+
+const UploadImage = (): JSX.Element => {
+  const [imageFile, setImageFile] = useState<string>('');
+
+  function convertFile (files: FileList | null): void {
+    if (files != null) {
+      const fileRef = files[0];
+      const fileType: string = fileRef.type;
+      console.log('This file upload is of type:', fileType);
+      const reader = new FileReader();
+      reader.readAsBinaryString(fileRef);
+      reader.onload = (ev: any) => {
+        setImageFile(`data:${fileType};base64,${btoa(ev.target.result)}`);
+      };
+    } else {
+      console.log('error');
+    }
+  }
+
+  return (
+    <div className="App">
+      <h2 className="text-sm font-semibold text-gray-600">
+        Choose an Image to Upload
+        </h2>
+        <br></br>
+        <div className="flex pl-5 text-sm font-semibold text-gray-600">
+          <form onSubmit={(e: React.FormEvent<HTMLFormElement>) => {
+            e.preventDefault();
+            console.log({ imageFile });
+            alert("here you'd submit the form using\n the filebase64 like any other field");
+          }}>
```

Comment:
As a good practice, you should extract this to a function outside the html section. It clutters the code.

Reply:
Also add a TODO to add the correct code logic in the future.
```
// TODO: todo message
```
and open an Issue on GitHub so that we can keep track of this